arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1069,6 +1069,14 @@
 				<key>storyboardName</key>
 				<string>CreateSymbolStylesFromWebStyles</string>
 			</dict>
+			<dict>
+				<key>displayName</key>
+				<string>Set up location-driven Geotriggers</string>
```

Comment:
I'll chat with the team to confirm 🤔

Reply:
Team thoughts:

- `Geotriggers` (plural with `s`) is a proper noun, so it should be capitalized in title or description.
- `geotrigger` is kinda a plain word when it is used in a phrase like `fence geotrigger` or `geotrigger feed`. It is also a countable noun so it can be `geotriggers` in a sentence

My thought:

- Maybe there should be a `Geotriggers` in the tags, but it's not included in the original design doc.

Reply:
Update:

Docs and Luke suggested not to capitalize it in the title, so I'll fix this. `common-samples/pull/2994`

Reply:
Now there shouldn't be capitalized proper noun `Geotriggers` in this sample.